apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2/BigQueryJob.cs

```diff
@@ -23,7 +23,7 @@
 namespace Google.Cloud.BigQuery.V2
 {
     /// <summary>
-    /// A dataset within BigQuery.
+    /// An asynchronous task within BigQuery.
```

Comment:
I'd rather use "job" here than task - asynchronous task sounds like it's a `Task<T>`.